plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java

```diff
@@ -955,7 +955,9 @@ public OrganizeImportsResult organizeImports(OrganizeImportParams organizeImport
    */
   public JavaCoreOptions getJavaCoreOptions(List<String> filter) {
     Type type = new TypeToken<JavaCoreOptions>() {}.getType();
-    return doGetOne(Commands.GET_JAVA_CORE_OPTIONS_СOMMAND, new ArrayList<>(filter), type);
+    JavaCoreOptions result =
+        doGetOne(Commands.GET_JAVA_CORE_OPTIONS_СOMMAND, new ArrayList<>(filter), type);
+    return result == null ? new JavaCoreOptions() : result;
```

Comment:
Why do we need the null check? Shoudn't this return a result or throw an exception?

Reply:
It's better to return empty map of java compiler options than null, if jdt.ls wasn't initialized